workflows/emulation/subset_training_data.py

```diff
@@ -0,0 +1,105 @@
+import argparse
```

Comment:
Script that subsets and saves smaller netcdf files from the full training data batches.